scripts/analysis/build_signalized_crossing_failure_pack_issue_2754.py

```
@@ -0,0 +1,366 @@
+        "schema_version": "signalized_crossing_failure_pack.v1",
+        "negative_control": False,
+        "status": "failures_present",
+        "cases": cases,
+    }
+
+
+def load_trace_with_fallback(path: Path) -> SimulationTraceExport:
+    """Load a simulation trace export, falling back to a raw JSON load if validation fails."""
+    try:
+        return load_simulation_trace_export(path)
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)